ajax/libs/timelinejs/package.json

```diff
@@ -1,29 +1,29 @@
 {
   "name": "timelinejs",
   "filename": "js/timeline-min.js",
-  "version": "2.36.0",
+  "version": "3.4.7",
   "description": "TimelineJS is an open-source tool that enables you to build visually-rich interactive timelines and is available in 40 languages.",
-  "homepage": "http://timeline.knightlab.com/",
+  "homepage": "https://timeline.knightlab.com/",
   "keywords": [
     "timeline"
   ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/NUKnightLab/TimelineJS3.git"
+  },
   "autoupdate": {
     "source": "git",
-    "target": "git://github.com/NUKnightLab/TimelineJS.git",
+    "target": "git://github.com/NUKnightLab/TimelineJS3.git",
     "fileMap": [
       {
-        "basePath": "build",
+        "basePath": "compiled",
         "files": [
           "css/**/*",
           "js/**/*"
         ]
       }
     ]
   },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/NUKnightLab/TimelineJS.git"
-  },
```

Comment:
Yes let's move it back, we should not touch unrelated things, the less diff the better!